src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -380,7 +380,9 @@ private static DynamicJsonValue GetSubscriptionConnectionDJV(SubscriptionConnect
                 [nameof(SubscriptionConnection.ClientUri)] = x.ClientUri,
                 [nameof(SubscriptionConnection.Strategy)] = x.Strategy,
                 [nameof(SubscriptionConnection.Stats)] = GetConnectionStatsDJV(x.Stats),
-                [nameof(SubscriptionConnection.ConnectionException)] = x.ConnectionException?.Message
+                [nameof(SubscriptionConnection.ConnectionException)] = x.ConnectionException?.Message,
+                ["TCPConnectionStats"] = x.TcpConnection.GetConnectionStats(),
+                ["StatusSubscriptions"] = new DynamicJsonArray(x.GetRecentStatusDescriptions())
```

Comment:
The method here is `GetRecentStatusDescriptions` but the property is `StatusSubscriptions`. Is this valid?

Reply:
NP, I'll change this property to be a public readonly and create a copy of it upon consumption